packages/cli/src/loaders/tmx.ts

```diff
@@ -0,0 +1,100 @@
+import { ILoader } from "./_types";
+import { createLoader } from './_utils';
+import { DOMParser, XMLSerializer } from 'xmldom';
+import { Builder, parseStringPromise } from "xml2js";
+
+
+function normalizeLocale(locale: string): string {
+  
+  const localeMapping: Record<string, string> = {
+    ru: "ru",
+    fr: "fr",
+    es: "es-ES",
+    de: "de",
+    "zh-Hans": "zh-Hans",
+    ko: "ko",
+    ja: "ja",
+    it: "it",
+    ar: "ar"
+  };
+  if (!localeMapping[locale]) {
+    console.warn(`Locale "${locale}" is not recognized. Using original value.`);
+  }
+  
+  return localeMapping[locale] || locale;
+}
```

Comment:
I'm not sure this is needed / this is the best approach here...

maybe some helper functions from @replexica/spec could help?